### DISCUSSION TITLE
Is it necessary to have an officially registered organization to create an organization account?

### DISCUSSION THREAD

**Discussion by Stefman87** [community/community#140164] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello. Could anyone say if it is necessary to have an officially registered organization to create an organization account? For example, can our team create an organization account taking into consideration that we are NOT an organization we are simply a group of developers who work together?

**Comment by dalidoshx** [comment]
Hi @Stefman87 ,

You don’t need to be an officially registered organization to set up an organization account. I successfully created one yesterday (I was just testing things out). If you have any other questions, feel free to ask!

Best,
Adrian

**Reply by Stefman87** [reply]
Yes, I can create an organization account too. The question is if it won't be deleted/blocked/banned in the future because the created account doesn't belong to a real organization.

**Reply by dalidoshx** [reply]
I think you'll be fine if you don't use it for any harmful purposes. That's what organizations are for - to start small and see where it takes you. Who knows, you might just end up with a big one in the end and then you might need to register it! :)

For more information please check this link below:  https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations

**Comment by AidenJamesYt** [comment]
It is not required.

-FSS Team

[Fox Studios - Scratch | GitHub Repository](https://github.com/Fox-Studios-Scratch)
[Need Help? Join Our Discord!](https://sites.google.com/view/fox-studios-scratch/discord)
